.github/workflows/deployment.yml

```diff
@@ -44,20 +44,11 @@ jobs:
           oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_DEV_TOKEN }}"
           bash openshift/scripts/oc_provision_backup_s3_postgres_cronjob.sh ${SUFFIX} apply
 
-      - name: Prepare MariaDB Backup Deployment Config
-        shell: bash
-        run: |
-          oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_DEV_TOKEN }}"
-          EPHEMERAL_STORAGE=True bash openshift/scripts/oc_provision_backup_mariadb.sh ${SUFFIX} apply
-
-      - name: Prepare MariaDB Backup Cronjob
-        shell: bash
-        run: |
-          oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_DEV_TOKEN }}"
-          EPHEMERAL_STORAGE=True bash openshift/scripts/oc_provision_backup_mariadb_cronjob.sh ${SUFFIX} apply
-
-  build-api-image:
-    name: Build API Image
+  build-web-image:
+    # Declared ahead of build-api-image ; it runs slightly slower than the api build, and putting
+    # it here increases the odds that it get's started 1st, so api and web are slightly more likely
```

Comment:
I'm not sure that this assumption of the order of job execution is true, but the only documentation I can find on job ordering is if you put them in the same concurrency group: https://docs.github.com/en/actions/using-jobs/using-concurrency. I think the safest assumption is any job can be picked up at any time if it has no dependencies.

Reply:
@conbrad I think you're right re. order of job execution. I don't think there's any kind of guarantee. I have however noticed that it seems to go from top to bottom. If it is top to bottom, then this will be slightly faster, if not, all is well.